GCDWebServer.xcodeproj/project.pbxproj

```diff
@@ -861,13 +861,14 @@
 					};
 					CEE28CEE1AE0051F00F4023C = {
 						CreatedOnToolsVersion = 6.3;
-						DevelopmentTeam = 88W3E55T4B;
+						ProvisioningStyle = Manual;
```

Comment:
There are 3 targets in the project: Mac, iOS, tvOS. I think automatic signing should work for iOS and tvOS, no? So 4 changes.